packages/config/types/render.d.ts

```diff
@@ -23,5 +23,6 @@ export interface NuxtConfigurationRender {
   http2?: any // TBD
   resourceHints?: boolean
   ssr?: boolean
+  ssrLog?: true | false | 'collapsed'
```

Comment:
```suggestion
  ssrLog?: boolean | 'collapsed'
```

Dunno if this suggestion is enough relevant, in any case it's typed correctly 😁